chirp/src/db/content.rs

```diff
@@ -23,11 +23,13 @@ pub fn content_title_clean(mut title: String) -> String {
 pub fn db_map_content_query<P: Params>(
     s: &mut Statement,
     p: P,
+    clean_titles: Option<bool>,
```

Comment:
In hindsight the `Option` is useless here. The parameter still has to be passed in.